src/watcher.js

```diff
@@ -31,8 +31,8 @@ Watcher.prototype = {
         }
 
         // Create an individual watcher for this file.
-        streams[bundleFile] = gulpWatch(bundleFile, function (bundleFile) {
-          bundleFile = bundleFile.path;
+        streams[bundleFile] = gulpWatch(bundleFile, function (bundleFileVinyl) {
+          bundleFile = bundleFileVinyl.path;
```

Comment:
Not sure that would work because of the asynchrony of `gulpWatch`. This is fine, though. It doesn't leave room for any side-effects.